dangerfile.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const commits = danger.github.commits;
+const pr = danger.github.pr;
+
+// Fail when there are fixup/squash commits in the PR
+commits.forEach(commit => {
+    const commitMessage = commit.commit.message;
+    if (commitMessage.indexOf('fixup!') !== -1 || commitMessage.indexOf('squash!') !== -1) {
```

Comment:
Use can use `String.prototype.includes` 😉

```js
if (commitMessage.includes('fixup!') || commitMessage.includes('squash!')) {
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

Reply:
:+1: Thanks for pointing out, I ended up using `startsWith()`